### PR TITLE
Created the version 7.4 including bugfixes as well as update documentation.

### PR DESCRIPTION
**Pull request by abdel792**
#### This PR includes the following additions:

* Creation of version 7.4 including bugfixes as well as documentation update;
* Make the repo ready to receive L10N updates from SRT repo.

### CODE REVIEW

File: addon/globalPlugins/Weather_Plus/__init__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 # Released under GPL 2
 #This file is covered by the GNU General Public License.
 #See the file COPYING for more details.
-#Version 7.3
+#Version 7.4
 # Python 2 and 3 compatible
 
 import os,sys, winsound, config, globalVars, ssl, json
```
Comment (nvdaes) [inline addon/globalPlugins/Weather_Plus/__init__.py]:
```suggestion
import os, sys, winsound, config, globalVars, ssl, json
```

**Review by nvdaes — COMMENTED** [review]
@abdel792, you may also want to include comments for translators in code. Also, in the readme, though I haven't read it completely, you may perform the following improvements:
- Use checkbox consistently, replacing chack box (used sometimes, not always) with checkbox.
- After a dot the next character is not always capitalized.

**Comment by nvdaes** [comment]
Sorry, I made a mistake here: > • Use checkbox consistently, replacing chack box (used sometimes, not always) with checkbox.
I meant chack, not chack.

**Review by nvdaes — APPROVED** [review]
This PR contains appropiate improvements, so I will approve it. Anyway, more refinements in documentation and other improvements maybe done progressively.
Thanks for this work.

**Comment by abdel792** [comment]
Thanks @nvdaes,

I just merged this merge into the original repo.

Thanks again for the help you provide to the add-on developers for NVDA.

Kind regards.

**Pull request merged**